css/style.css

```diff
@@ -0,0 +1,1248 @@
+
+
+  [title="portfolio navigation"] {
+    display: -webkit-flex;
+    display: flex;
+    -webkit-flex-basis: 100%;
+    flex-basis: 100%;
+    -webkit-justify-content: center;
+    justify-content: center;
+    margin-top: 1em;
+    margin-bottom: 1.777em;
+  }
+
+  [title="portfolio-categories"] li {
+    display: inline-block;
+  }
+
+  .portfolio-menu-item {
+    font-size: 1rem;
+  }
+}
+
+
+/*
+PORTFOLIO ITEMS
+*/
+
+.portfolio-item {
+  position: relative;
+  max-width: 300px;
+  margin-top: .5em;
+  margin-bottom: 2em;
+  text-align: center;
+}
+
+.portfolio-opacity {
+  opacity: .7;
+}
+
+.portfolio-link-hover:hover {
+  display: block;
+  background: #000;
+}
+
+.portfolio-link-hover:hover:after {
+  position: absolute;
+  top: 30%;
+  left: 42%;
+  width: 1.777em;
+  height: 1.777em;
+  padding-top: .4em;
+  content: "\e608";
+  font-family: 'ico-fonts';
+  font-size: 1.6em;
+  text-align: center;
+  color: #fff;
+  background-color: #fd6632;
+  border-radius: 50%;
+}
+
+.portfolio-caption-header {
+  padding: .5em 0;
+  font-weight: bold;
+  font-size: 1.333em;
+}
+
+.portfolio-category {
+  color: #7c8387;
+}
+
+/*
+ABOUT
+*/
+
+.about, .about p {
+  color: #b9bdbf;
+}
+
+.about {
+  background-color: #252629;
+  padding: 4em 1em 1em;
+}
+
+.about-header, .skills-header {
+  color: #fff;
+  text-decoration: underline;
+}
+
+.about-header {
+  font-size: 2.4em;
+  margin-bottom: 0.5em;
+}
+
+.about p {
+  display: inline-block;
+  line-height: 1.5em;
+  margin-bottom: 1em;
+}
+
+.skills-header {
+  font-size: 1.7em;
+  margin-bottom: 0.8em;
+}
+
+.skills li {
+  margin-bottom: 1em;
+}
+
+.progress-header {
+  text-align: left;
+  margin-bottom: 0.3em;
+}
+
+progress[value]::-webkit-progress-bar {
+  background-color: #3a3a3a;
+}
+
+progress[value]::-webkit-progress-value {
+  background-color: #fd6632;
+}
+
+.join-us-header {
+  font-weight: normal;
+  font-size: 1.5em;
+  margin-bottom: 1em;
+}
+
+.contact-us-button {
+  font-weight: bold;
+  padding: 0.9em 3.5em;
+}
+
+.about>aside {
+  margin-bottom: 1em;
+}
+
+@media screen and (min-width: 50em) {
+  .about-header, .skills-header {
+    text-decoration: none;
+  }
+
+  .about-header {
+    margin: 0 12% 1em;
+    background-size: calc(50% - 2em);
+  }
+
+  .about p {
+    width: 49.5%;
+    padding: 0 0.5em;
+    margin-bottom: 6em;
+  }
+
+  .skills {
+    margin-bottom: 6em;
+  }
+
+  .skills-header {
+    margin: 0 19.25% 1.5em;
+    background-size: calc(50% - 2.25em);
+  }
+
+  .skills li {
+    margin-bottom: 1.4em;
+  }
+
+  progress[value] {
+    height: 1.8em;
+  }
+
+  .join-us-header {
+    margin-bottom: 1.5em;
+  }
+}
+
+/*
+TEAM
+*/
+
+.team, .clients {
+  display: -webkit-box;
+  display: -webkit-flex;
+  display: flex;
+  -webkit-flex-wrap: wrap;
+  -ms-flex-wrap: wrap;
+  flex-wrap: wrap;
+}
+
+.team {
+  -webkit-justify-content: space-around;
+  justify-content: space-around;
+  padding-top: 2.5em;
+}
+
+.team-section-header {
+  padding: 0.7em 0;
```

Comment:
Be consistent with whether or not you use a leading zero for decimal values. Most of the stylesheet has it without, but this one has it with the `0`. Generally, it's a good idea to use the leading `0`, since all other values _also_ have a value before the decimal, and it makes it easier to scan.